src/main/java/io/spaship/operator/service/k8s/Operator.java

```diff
@@ -37,414 +36,413 @@
+                return;
+            }
+            eventManager.queue(eb.build());
+            LOG.debug("network management and rbac policy installed successfully in namespace {} ",
+                    environment.getNameSpace());
+        };
+    }
+
+    public String environmentSidecarUrl(Environment environment) {
+        String serviceName = "svc"
+                .concat("-")
+                .concat(appInstance)
+                .concat("-")
+                .concat(environment.getWebsiteName().toLowerCase())
+                .concat("-")
+                .concat(environment.getName().toLowerCase());
+        LOG.debug("computed service name is {}", serviceName);
+        var svc = ocClient.services().inNamespace(environment.getNameSpace()).withName(serviceName).get();
+
+        String clusterIP = svc.getSpec().getClusterIP();
+        var svcPort = svc.getSpec().getPorts().stream()
+                .filter(port -> Objects.equals(port.getName(), "http-api"))
+                .findFirst()
+                .orElseThrow(() -> new ResourceNotFoundException("service port not found"))
+                .getPort();
+        LOG.debug("computed service port is {}, clusterIp is {}", svcPort, clusterIP);
+
+        String sideCarSvcUrl = "tcp://".concat(clusterIP).concat(":").concat(String.valueOf(svcPort));
+        LOG.debug("computed sidecar service url is {}", sideCarSvcUrl);
+
+        return sideCarSvcUrl;
+
     }
-    if (pods.isEmpty()) {
-      LOG.info("no pod found with the following details {}", labels);
-      return isAvailable;
+
+    Map<String, String> searchCriteriaLabel(Environment environment) {
+        return Map.of(MANAGED_BY, SPASHIP,
+                WEBSITE, environment.getWebsiteName().toLowerCase(),
+                ENVIRONMENT, environment.getName().toLowerCase());
+    }
+
+    private OperationResponse applyDeleteResourceList(Environment environment, KubernetesList resourceList) {
+        LOG.debug("applying delete on resources");
+        // Should we delete the pvs as well? will that be a good idea?
+        var isDeleted = ocClient.resourceList(resourceList).inNamespace(environment.getNameSpace()).delete();
+        environment.setOperationPerformed(true);
+        var or = OperationResponse.builder().environment(environment)
+                .sideCarServiceUrl("NA")
+                .originatedFrom(this.getClass().toString());
+        or.status(3);
+        if (isDeleted.isEmpty())
+            or.status(0).errorMessage("unable to delete the resources");
+        return or.build();
+    }
+
+    private KubernetesList buildK8sResourceList(Environment environment) {
+        LOG.info("inide buildK8sResourceList the environment value is {}", environment);
+        Map<String, String> templateParameters = Map.of(
+                "WEBSITE", environment.getWebsiteName().toLowerCase(),
+                // "TRACE_ID", environment.getTraceID().toString().toLowerCase(),
```

Comment:
// "TRACE_ID", environment.getTraceID().toString().toLowerCase(),\
Remove this commented line, if not required in future